app/routes/errors.py

```diff
@@ -8,14 +9,29 @@
 from app.authentication.no_token_exception import NoTokenException
 from app.globals import get_metadata
 from app.helpers.template_helper import render_template
-from app.libs.utils import convert_tx_id_for_boxes
 from app.submitter.submission_failed import SubmissionFailedException
 
 
 logger = get_logger()
 
 errors_blueprint = Blueprint('errors', __name__)
 
+ERROR_MAP = {
+    500: _('An error has occurred'),
+    404: _('Page not found'),
+    403: _('Authorisation refused'),
+}
+
+DEFAULT_ERROR = 500
+
+
+def get_error_title(status_code):
+    return (
+        ERROR_MAP.get(status_code, ERROR_MAP.get(DEFAULT_ERROR))
+        + ' - '
+        + _('Office for National Statistics')
+    )
+
```

Comment:
We could remove the error map and move the strings to the templates, setting `page_title` as we do for example in `signed-out.html`.